seckill-command-service/src/main/java/io/servicecomb/poc/demo/seckill/SecKillPromotionBootstrap.java

```diff
@@ -53,15 +53,30 @@ public SecKillPromotionBootstrap(
 
   public void run() {
     CompletableFuture.runAsync(() -> {
```

Comment:
this class seems to run in background forever. it's better to use scheduled task in this case